raft/src/main/java/org/apache/kafka/raft/KafkaNetworkChannel.java

```diff
@@ -185,6 +187,10 @@ static AbstractRequest.Builder<? extends AbstractRequest> buildRequest(ApiMessag
             return new FetchRequest.SimpleBuilder((FetchRequestData) requestData);
         if (requestData instanceof FetchSnapshotRequestData)
             return new FetchSnapshotRequest.Builder((FetchSnapshotRequestData) requestData);
+        if (requestData instanceof ApiVersionsRequestData)
+            return new ApiVersionsRequest.Builder((ApiVersionsRequestData) requestData,
+                ApiKeys.API_VERSIONS.oldestVersion(),
+                ApiKeys.API_VERSIONS.latestVersion());
```

Comment:
why do we need to specify these versions explicitly? is it because we need `enableUnstableLastVersion==true`?

Reply:
The reason is more mundane than that. I didn't want to have a lot of constructors, so I decided to have:

- no-args
- required-version
- data, min-version, max-version

I could create lots of permutations, like:

- no-args 
- required-version
-  data
- data, required-version
- data, min-version, max-version

but 3 constructors seems like enough?